include/swift/Driver/Driver.h

```diff
@@ -165,6 +165,9 @@ class Driver {
   /// The original path to the executable.
   std::string DriverExecutable;
 
+  // Extra args to pass to the driver executable
+  SmallVector<std::string, 10> DriverExecutableArgs;
```

Comment:
This could probably be something like 2 or 4 with nothing lost in practice. (Yeah, I know there's only one of these, but still.)